test/module/shared_model/validators/field_validator_test.cpp

```diff
@@ -236,6 +263,51 @@ class FieldValidatorTest : public ValidatorsTest {
        false,
        "Amount must be greater than 0, passed value: 0"}};
 
+  std::string invalidPagerLimitMessage(uint16_t value) {
+    return (boost::format("Pager limit must be greater than 0 and less than or "
+                          "equal to 100, "
+                          "passed value: %d")
+            % value)
+        .str();
+  }
+
+  std::vector<FieldTestCase> pager_test_cases{
```

Comment:
Is it method passed clnang-format?

Reply:
Yes, though maybe something looks weird.